src/SqlSquared/Signature.purs

```diff
@@ -878,16 +895,19 @@ genFunctionDecl n = do
   args ← L.foldM foldFn L.Nil $ L.range 0 len
   pure $ FunctionDecl { ident, args, body: n - 1 }
 
-genImport ∷ ∀ m a. Gen.MonadGen m ⇒ m (SqlDeclF a)
-genImport = Import <$> genIdent
+genImport ∷ ∀ m a. Gen.MonadGen m ⇒ MonadRec m ⇒ m (SqlDeclF a)
+genImport = map Import
+  $ Gen.oneOf
+  $ (Pt.unsandbox >>> E.Left <$> PtGen.genAbsDirPath)
```

Comment:
Flipped flow. `Pt.unsandbox >>> E.Left` left-to-right, `<$>` right-to-left. It's hard to follow, would you mind switching to `E.Left <<< Pt.unsandbox <$> PtGen.genAbsDirPath` or `Pt.genAbsDirPath <#> Pt.unsandbox >>> E.Left`?